v2/tools/generator/internal/astmodel/known_locals_set.go

```diff
@@ -9,9 +9,9 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/gobuffalo/flect"
-
 	"github.com/Azure/azure-service-operator/v2/internal/set"
+
```

Comment:
minor: I think can remove this space? IIRC all of the `github.com/Azure/azure-service-operator/v2` imports should go together in an ordered block